src/main/kotlin/com/intershop/gradle/icm/docker/utils/Configuration.kt

```diff
@@ -174,4 +174,7 @@ object Configuration {
     const val NGINX_PRIVATEKEY_FILENAME_VALUE = "privkey.pem"
 
     const val INTERSHOP_ENVIRONMENT_PREFIX = "intershop.environment."
+
+    const val REDIS_PORT = "redis.port"
```

Comment:
How is this configuration "submitted" to the as-container. There's already code, that does something like that e.g. use the host/port of the solr-container as an input for the as-container.

Reply:
As discussed, I removed the configuration option of the redis port entirely, since it is not easily possible to also respect a differently configured port on the icm-as side.